src/main/java/com/conveyal/datatools/manager/controllers/api/FeedSourceController.java

```diff
@@ -332,7 +344,40 @@ public static FeedSource checkFeedSourcePermissions(Request req, FeedSource feed
             // Throw halt if user not authorized.
             logMessageAndHalt(req, 403, "User not authorized to perform action on feed source");
         }
+
+
         // If we make it here, user has permission and the requested feed source is valid.
+        // This final step removes labels the user can't view
+        return cleanFeedSourceLabels(feedSource, userProfile);
+    }
+
+    /** Determines whether a change to a feed source is significant enough that it warrants sending a notification
+     *
+     * @param formerFeedSource  A feed source object, without new changes
+     * @param updatedFeedSource A feed source object, with new changes
+     * @return                  A boolean value indicating if the updated feed source is changed enough to warrant sending a notification.
+     */
+    private static boolean shouldNotifyUsersOnFeedUpdated(FeedSource formerFeedSource, FeedSource updatedFeedSource) {
+        return
+                // If only labels have changed, don't send out an email
+                formerFeedSource.equalsExceptLabels(updatedFeedSource);
+    }
```

Comment:
Just double-checking here, is this "do not notify of feed source changes when only labels change" a kind of functionality we want to enable? Perhaps some users might want to know if a label is added to a feed source.

Reply:
I think the number of emails this would generate would be really noisy. There are some other examples throughout the code of this feature being disabled, especially since the email doesn't say which part of a feed source was updated, it just says "feed source property changed"
<img width="1552" alt="Screen Shot 2021-08-03 at 8 06 34 AM" src="https://user-images.githubusercontent.com/86619099/128012579-85336399-53b5-4cc5-a5a2-54be937dcb6a.png">

Reply:
Yes, I believe this is noisy and I would do this, but I'm more just wanting to double check if a client is asking for this functionality to be enabled/disabled.

Reply:
I checked internally with the team and am ok marking this resolved.